.github/workflows/sync.yml

```
@@ -91,8 +91,8 @@ jobs:
           RCLONE_CONF: ${{ secrets.RCLONE_CONF }}
           RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS: ${{ secrets.RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS }}
         with:
-          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /CMIP/{NCAR,CNRM-CERFACS,IPSL,CCCma,MPI-M,NCC,EC-Earth-Consortium,NASA-GISS,NOAA-GFDL,BCC,AWI,CSIRO,SNU,CSIRO-ARCCSS}/**' --filter '+ /CMIP/**' --filter '- *' --transfers 15 --checkers 65"
-  # ScenarioMIP/CCCma only
+          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /CMIP6/CMIP/{NCAR,CNRM-CERFACS,IPSL,CCCma,MPI-M,NCC,EC-Earth-Consortium,NASA-GISS,NOAA-GFDL,BCC,AWI,CSIRO,SNU,CSIRO-ARCCSS}/**' --filter '+ /CMIP/**' --filter '- *' --transfers 15 --checkers 65"
```

```suggestion
          args: "sync -c -v gs:cmip6 s3:cmip6-pds --filter '- /CMIP6/CMIP/{NCAR,CNRM-CERFACS,IPSL,CCCma,MPI-M,NCC,EC-Earth-Consortium,NASA-GISS,NOAA-GFDL,BCC,AWI,CSIRO,SNU,CSIRO-ARCCSS}/**' --filter '+ /CMIP6/CMIP/**' --filter '- *' --transfers 15 --checkers 65"
```

Slight tweak to filter

---

good catch!